src/fitnesse/slim/MethodExecutionResult.java

```diff
@@ -51,13 +57,36 @@ public boolean hasResult() {
     }
   }
 
+  public static class InvalidParameters extends MethodExecutionResult {
+
+	    private final String methodName;
+	    private final Exception conversionException;
+
+	    public InvalidParameters(String methodName, Exception conversonException) {
+	      super(null, null);
+	      this.methodName = methodName;
+	      this.conversionException = conversonException;
+	    }
+
+	    @Override
+	    public Object returnValue() {
+	      throw new SlimError(SlimError.extractSlimMessage(conversionException.getMessage())+ "\nTryed to invoke: " + methodName, conversionException, "", true);	
```

Comment:
```suggestion
	      throw new SlimError(SlimError.extractSlimMessage(conversionException.getMessage())+ "\nTried to invoke: " + methodName, conversionException, "", true);	
```
This also impact so unit tests